src/ModelingAlgorithms/TKTopAlgo/BRepTopAdaptor/BRepTopAdaptor_FClass2d.cxx

```diff
@@ -168,30 +168,38 @@ BRepTopAdaptor_FClass2d::BRepTopAdaptor_FClass2d(const TopoDS_Face&  aFace,
         if (Abs(plbid - pfbid) < 1.e-9)
           continue;
 
-        // if(degenerated==Standard_False)
-        //   C3d.Initialize(edge,Face);
-
         //-- Check cases when it was forgotten to code degenerated :  PRO17410 (janv 99)
         if (degenerated == Standard_False)
         {
           C3d.Initialize(edge, Face);
-          du          = (plbid - pfbid) * 0.1;
```

Comment:
u, du Are no longer used? Can we move out from the highest scope?
It looks like not used at the beginning and can be moved.